### PR TITLE
Remove outdated links from /learn

### PR DESCRIPTION
**Pull request by frankharkins**
## Changes

Some links on qiskit.org/learn are outdated and should be removed.

- "Bonus: Level sandbox" (and others) are actually a result of Platypus
  incorrectly splitting pages from notebook headers. The correct link is "Hello
  Qiskit" (added in #3129).

- "Widgets demonstration" shows widgets created for the old platform
  (qiskit-community/qiskit-textbook), that don't all work on Platypus. This
  page has been removed.

## Implementation details

This PR builds on #3129 and is a draft until #3129 is merged. I'll then re-test
and fix any merge conflicts.

### CODE REVIEW

**Comment by techtolentino** [comment]
@frankharkins - FYI https://github.com/Qiskit/qiskit.org/pull/3129 has been merged 

Thanks!

File: pages/learn/course/games-and-demos.vue [hunk 1]
```diff
@@ -40,7 +40,7 @@ const headerDescription = [
 const headerImg = "/images/learn/course/games-and-demos/games-and-demos.png";
 
 const startLearningCTA: GeneralLink = {
-  url: GamesAndDemos.BonusLevelSandbox,
+  url: GamesAndDemos.EstimatingPiUsingQPEA,
```
Comment (frankharkins) [inline pages/learn/course/games-and-demos.vue:43]:
```suggestion
  url: GamesAndDemos.HelloQiskit,
```

**Review by techtolentino — CHANGES_REQUESTED** [review]
Hey @frankharkins - thanks for the PR... 

Please remove the unused assets that were associated w/ the outdated links:

- [bonus-level-make-your-own-puzzles.png](https://github.com/Qiskit/qiskit.org/blob/fh-remove-learn-links/public/images/learn/course/games-and-demos/bonus-level-make-your-own-puzzles.png)
- [bonus-level-sandbox.png](https://github.com/Qiskit/qiskit.org/blob/fh-remove-learn-links/public/images/learn/course/games-and-demos/bonus-level-sandbox.png)
- [widgets-demonstration.png](https://github.com/Qiskit/qiskit.org/blob/fh-remove-learn-links/public/images/learn/course/games-and-demos/widgets-demonstration.png)

And since it's associated w/ this clean-up effort, and no longer used, can you also delete the following one as well

- [level-5-proving-uniqueness-of-quantum-variables.png](https://github.com/Qiskit/qiskit.org/blob/fh-remove-learn-links/public/images/learn/course/games-and-demos/level-5-proving-uniqueness-of-quantum-variables.png)

Once that's done, I'll approve.

Thanks!

**Comment by frankharkins** [comment]
Thanks! Sorry I didn't get round to removing the images in time.

**Pull request merged**